homeassistant/components/fritzbox/config_flow.py

```diff
@@ -120,6 +121,9 @@ async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowRes
         assert isinstance(host, str)
         self.context[CONF_HOST] = host
 
+        if ipaddress.ip_address(host).is_link_local:
```

Comment:
This can also be an IPv4 link local address? [docs](https://docs.python.org/3/library/ipaddress.html#ipaddress.IPv4Address.is_link_local)

Reply:
you are right - i've added an additional check for the ip version